cmd/network_chain_revert_launch.go

```diff
@@ -12,10 +12,19 @@ import (
 // to revert a launched chain.
 func NewNetworkChainRevertLaunch() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "revert-launch [launch-id]",
-		Short: "Revert launch of a network",
-		Args:  cobra.ExactArgs(1),
-		RunE:  networkChainRevertLaunchHandler,
+		Use: "revert-launch [launch-id]",
+		Short: `The revert launch command cancels the previously scheduled launch of a chain.
```

Comment:
```suggestion
		Short: `The revert launch command reverts the previously scheduled launch of a chain.
```
If think as per the command name, reverting is the right wording. Since this command is executed after the genesis time is reached. So the launch had been executed in the eyes of the Ignite chain (but in theory failed in the real world) but this action is reverted
Cancel would mean the action is cancel before being completed